javascript/computer_science/recursive_methods.md

```diff
@@ -40,7 +40,7 @@ This section contains a general overview of topics that you will learn in this l
 
 <div class="lesson-note lesson-note--warning" markdown="1">
 
-The solution for "Question 6: Search JS object" is incorrect. See the [corrected solution](https://gist.github.com/JoshDevHub/b00125f483d4a1ecc257eaa030916973) after you solve it.
+The solution for "Question 6: Search JS object" is incomplete. See the [corrected solution](https://gist.github.com/JoshDevHub/b00125f483d4a1ecc257eaa030916973) after you solve it.
```

Comment:
No issue with your change, just thought while we're at it, the current link text "corrected solution" isn't descriptive enough on its own for what it actually refers to, so would ideally be made more accessible with something like the following:

```suggestion
The solution for "Question 6: Search JS object" is incomplete. See [a more accurate solution for `contains()`](https://gist.github.com/JoshDevHub/b00125f483d4a1ecc257eaa030916973) after you solve it.
```